crates/parcel_filesystem/src/lib.rs

```diff
@@ -4,7 +4,6 @@ use std::path::PathBuf;
 
 use dashmap::DashMap;
 
-/// FileSystem implementation that delegates calls to a JS object
 pub mod js_delegate_file_system;
```

Comment:
If the idea is just to be able to compile ouside of napi you can just feature-gate this

Reply:
I did that initially but I feel it's better that the consumer defines their implementations that satisfy Parcel's defined interface/traits as there doesn't seem to be a benefit to this abstraction.

Essentially:

`parcel_core` should export the `FileSystem` trait 
`parcel_napi` should create an implementation that satisfies that trait

In future:
`parcel_wasm` (...etc) should have their own implementations

Reply:
Updated